qt/RpcQueue.cc

```diff
@@ -63,7 +63,9 @@ void RpcQueue::runNext(RpcResponseFuture const& response)
 {
     assert(!queue_.isEmpty());
```

Comment:
Should be change to `std::empty(queue_)`. Same at line 51.

Reply:
d'oh, thank you!